src/server/status/index.js

```diff
@@ -29,7 +34,8 @@ export default function (kbnServer, server, config) {
           build_snapshot: matchSnapshot.test(config.get('pkg.version'))
         },
         status: kbnServer.status.toJSON(),
-        metrics: kbnServer.metrics
+        metrics: kbnServer.metrics,
+        stats,
```

Comment:
Yea, that was my bad @tsullivan, a closer look and it seems easy enough to do in a first pass, despite my initial hesitations.  :)  I will give it a go!